freqtrade/commands/cli_options.py

```diff
@@ -345,6 +345,13 @@ def __init__(self, *args, **kwargs):
         type=check_int_positive,
         metavar='INT',
     ),
+    "new_pairs_days": Arg(
+        '--new-pairs-days',
+        help='Download data of new pairs for given number of days. Default: `%(default)s`.',
+        type=check_int_positive,
+        metavar='INT',
+        default=30,
```

Comment:
instead of defining the default here - we should rather move this to `constants.py` and add the default in the json validation.

This will allow providing this value via config as well - otherwise the "argument default" from the CLI will always overwrite what is in the config file ... .

Reply:
Oh ok! So this is why putting it in config didnt work

Reply:
yeah, argparse default arguments are good if you ONLY use argparse - but if you use a default, they'll ALWAYS have this value - so there's no way to determine if a user provided that explicitly, or if it's coming as default (at least not without going to parse the provided arguments manually, which defeats the purpose of using argparse ...)